lib/lib_rss.php

```diff
@@ -578,3 +578,18 @@ function validateShortcutList($shortcuts) {
 	}
 	return $shortcuts_ok;
 }
+
+function errorMessage($errorTitle, $error = '') {
+	$message=<<<MSG
```

Comment:
As you only return a string value, I would omit the use of a variable by using `return` directly:
```php
return <<<MSG
...
...
MSG;
```

Reply:
Huh, I didn't know return would do the equivalent of an echo?